examples/connext_dds/detect_samples_dropped/c++/droppedSamplesExample_publisher.cxx

```diff
@@ -14,8 +14,8 @@
 #include <stdio.h>
 #include <stdlib.h>
 
-#include "drop.h"
-#include "dropSupport.h"
+#include "droppedSamplesExample.h"
+#include "droppedSamplesExampleSupport.h"
 #include "ndds/ndds_cpp.h"
 #include "application.h"
```

Comment:
Has your clang-format file sorted include files like this? If it has, reorganize them however you want and do a pull from the remote repo, we've removed automatic sorting, because it was kind of weird.

Reply:
I have updated my repo with `https://github.com/rticommunity/rticonnextdds-examples`
What do you mean?